.github/workflows/publish.yml

```diff
@@ -1,7 +1,9 @@
 name: Build and Publish
 on:
   push:
-    branches: master
+    branches:
```

Comment:
To cover forks where people may prefer to use the branch name "main". I understand that that's likely to be a small subset of users (just myself right now, it would appear), but I don't see what it hurts. I can revert this back and change mine.